classes/eventobservers.php

```diff
@@ -209,4 +211,48 @@ private static function clear_collapse_memory($userid) {
             }
         }
     }
+
+    /**
+     * cohort member added event observer.
+     *
+     * @param \core\event\cohort_member_added $event The event.
+     */
+    public static function cohort_member_added(\core\event\cohort_member_added $event) {
+
+        self::update_cohort_revision_cache();
+    }
+
+    /**
+     * cohort member removed event observer.
+     *
+     * @param \core\event\cohort_member_removed $event The event.
+     */
+    public static function cohort_member_removed(\core\event\cohort_member_removed $event) {
+
+        self::update_cohort_revision_cache();
+    }
+
+    /**
+     * cohort deleted event observer.
+     *
+     * @param \core\event\cohort_deleted $event The event.
+     */
+    public static function cohort_deleted(\core\event\cohort_deleted $event) {
+
+        self::update_cohort_revision_cache();
+    }
+
+    /**
+     * Update cohort revision cache.
+     *
+     */
+    private static function update_cohort_revision_cache() {
+        try {
+            $cache = cache::make('local_boostnavigation', 'cohortversion');
+        } catch (Exception $e) {
```

Comment:
same here, not sure the catch adds much value

Reply:
Thank you Brendan !!

Removed try catch block. Please review.

Thanks,
Anu